crates/ruff_linter/resources/test/fixtures/pyflakes/F401_25__all/__init__.py

```diff
@@ -0,0 +1,52 @@
+'''__init__.py with __all__
+
+Unused stdlib and third party imports are unsafe removals
+
+Unused first party imports get added to __all__
+'''
+
+
+
+# stdlib
+
+
+
+import os # Ok: is used
+_ = os
+
+
+
+import argparse # Ok: is exported in __all__
+
+
+
+import sys # F401: remove unused
+
+
+
+# first-party
+
+
+
+from . import used # Ok: is used
+_ = used
+
+
+
+from . import aliased as aliased # Ok: is redundant alias
+
+
+
+from . import exported # Ok: is exported in __all__
+
+
+
+#from . import unused # F401: add to __all__
```

Comment:
Should we add comments to all of the empty helper files saying what they're used for?